third_party/nvfuser/csrc/transform_replay.h

```diff
@@ -124,21 +124,30 @@ class RootDomainMap;
 
 class TORCH_CUDA_CU_API TransformReplay {
  public:
-  // Replay producer as consumer, returns {producer, producer_compute_at_axis}.
+  // Replay producer as consumer, returns {producer,
+  // producer_compute_at_axis}.
+  //
+  // replay_resize indicates whether resize should be replayed or
+  // ignored. It is only replayed when replaying a producer for
+  // indexing.
   static std::pair<TensorDomain*, unsigned int> replayPasC(
       const TensorView* producer,
       const TensorView* consumer,
       int consumer_compute_at_axis,
-      bool replay_swizzle = false);
+      bool replay_swizzle = false,
+      bool replay_resize = false);
```

Comment:
This is not your fault, but in `TransformReplay`, we have `replay_resize`, but in `BestEffortReplay`, we have `skip_resize`. We should consider making them use the same name.

Reply:
Yes, I'll refactor them as a followup PR.